### ISSUE TITLE
Rule not applied when I think it should be.

### ISSUE THREAD

**Issue by willfinnigan** [connorcoley/rdchiral#18]
Hi,

Nice project.

I've got quite a simple rule I'd like to apply which has no chirality specified.

[#6:4][#8:6][#6:2](=[O:3])[#6:1] >> [#6:4][#6:2](=[O:3])[#6:1]

<img width="612" alt="Screenshot 2020-03-02 at 18 54 01" src="https://user-images.githubusercontent.com/15628614/75707679-39feec80-5cb7-11ea-8880-cce6ee4d1d1c.png">

This works fine in most cases, but not for this ring:

C[C@@H]1CCOC(=O)CC1

<img width="250" alt="Screenshot 2020-03-02 at 18 54 06" src="https://user-images.githubusercontent.com/15628614/75707726-5ac74200-5cb7-11ea-8efe-01e2965c1bc6.png">

However, rings one carbon bigger or one carbon smaller work fine.

Is it something to do with it being a symmetrical product?   The standard rdkit method works ok.

Thanks for any help!!

Best,
Will

**Comment by willfinnigan** [comment]
```
from rdchiral.main import rdchiralReaction, rdchiralRunText, rdchiralReactants, rdchiralRun

rxn = rdchiralReaction('[#6:4][#8:6][#6:2](=[O:3])[#6:1]>>[#6:4][#6:2](=[O:3])[#6:1]')
reactants = rdchiralReactants('C[C@@H]1CCOC(=O)CC1')
outcomes = rdchiralRun(rxn, reactants)

print(outcomes)
```

**Comment by thomasstruble** [comment]
You are correct, the outcomes from rdkit has stereochemistry when the symmetry of the product makes the molecule achiral. The lines in the code that deals with this are: https://github.com/connorcoley/rdchiral/blob/3b9ae0e02d1ec99eea1b953d5870091af664de4f/rdchiral/main.py#L498-L511

**Comment by connorcoley** [comment]
Hi Will -- Thanks for creating this issue. Viewed in the context of retrosynthesis, this looks like the desired behavior. We wouldn't be able to synthesize ```C[C@@H]1CCOC(=O)CC1``` through that ring expansion; we could only make a racemic mixture since we are starting from the achiral precursor.

If you're interested in changing this behavior, you could consider modifying the lines @thomasstruble referred you to. However, we always handle ambiguity in a manner that makes sense for retrosynthesis throughout the code, so I'd caution you against making just that single change

**Comment by willfinnigan** [comment]
Ah that makes sense! (sorry, biologist here)

However, enzymatically this appears to be possible.
https://onlinelibrary.wiley.com/doi/epdf/10.1002/cctc.201300604

I'll have a play with the change @thomasstruble suggested.

Thanks both for getting back.  Hopefully see you in Bristol next week.
